src/dotnet-mssql/README.md

```diff
@@ -7,7 +7,7 @@ Develop C# and .NET Core based applications. Includes all needed SDKs, extension
 
 | Options Id | Description | Type | Default Value |
 |-----|-----|-----|-----|
-| imageVariant | .NET version: | string | 8.0-bookworm |
+| imageVariant | .NET version: | string | 9.0-bookworm |
```

Comment:
Once this PR is merged, https://github.com/devcontainers/action will run automatically ; which opens a PR that updates the README file based on `devcontainer-feature.json` and `NOTES.md`.

Hence, can we revert the changes to this file to avoid merge conflicts? Thanks!